doc/developer-documentation/forgae-cli/node.md

```diff
@@ -8,7 +8,8 @@ forgae node
 
 ```
 
-The **node** command help developers run their local network on docker.
+The package is responsible for setting up a healthy local node. 
+The **node** command help developers run their local network, as well as local compiler on docker.
```

Comment:
I think here is missing the whole information about the compiler and the **- - compiler** option, if not excuse me. If it is missing you can check the current documentation